junit5-jupiter-starter-bazel/BUILD

```diff
@@ -0,0 +1,20 @@
+load("//:junit5.bzl", "java_junit5_test")
+
+java_library(
+    name = "junit5-jupiter-starter-bazel",
+    srcs = glob([
+        "src/main/java/**/*.java",

```

Comment:
@marcphilipp You guys comments conflict. I think @jbduncan's suggest is better. 

However it may confuse the new beginners for separating definitions into several individual files. I'd like to put them into the root BUILD file & use absolute accural file paths.

Reply:
> However it may confuse the new beginners for separating definitions into several individual files.

Hmm, that's a good point...

@hcoona Do you know, by any chance, if it's possible to have a root BUILD file that depends on the targets in the existing BUILD files in `src/main/java/com/example/project` and `src/test/java/com/example/project`?

If so, then that may be a good compromise.

@marcphilipp What do you think?

Reply:
It's possible to add a tar_pkg target relying on these 2 targets. But it's not quite meaningful.